host/src/lib/HelperFunctions.jsx

```diff
@@ -0,0 +1,114 @@
+import { isNullOrUndefined, GameSessionState } from "@righton/networking";
+/* 
+* counts all answers for current question using isChosen, for use in footer progress bar
+* @param {array} answerArray - array of answers for current question
+* @returns {number} count - number of answers for current question
+*/
+export const getTotalAnswers = (answerArray) => {
+  let count = 0;
+  if (answerArray) {
+    answerArray.forEach(answerCount => {
+      count = count + answerCount;
+    });
+    return count;
+  }
+  return count;
```

Comment:
ugh, totally. fixed

Reply:
update: this is actually counting the sum of the array, rather than the length. answerArray here is an array of length 4 that contains the # of answers for each selected value (A, B, C, D). so if 2 players answered A and two players answered C, answerArray would be [2, 0, 2, 0]. 

I still took a look at the original code and am using `reduce` to refine it a bit. It's now:
```
export const getTotalAnswers = (answerArray) => {
  return answerArray ? answerArray.reduce((accumulator, currentValue) => accumulator + currentValue, 0) : 0;
};
```